src/characters.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>ORCs - Online Role-playing Characters</title>
+    <title>List of characters</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
     <link rel="stylesheet" href="Stil.css">
```

Comment:
You should include jquery before bootstrap.js!
Remove one of them, you included 2 different version of it!